lib/team.rb

```diff
@@ -1,16 +1,16 @@
 class Team
   attr_reader :team_id,
-              :franchise_id,
-              :team_name,
+              :franchiseid,
+              :teamname,
               :abbreviation,
               :stadium,
               :link,
               :team_list
   
   def initialize(row, team_list)
     @team_id       = row[:team_id]
-    @franchise_id  = row[:franchise_id]
-    @team_name     = row[:team_name]
+    @franchiseid   = row[:franchiseid]
```

Comment:
Too late to make any adjustments, buuut I think technically following proper ruby convention would be:
```
@franchise_id = row[:franchiseid]
@team_name = row[:teamname]
```
I don't think we have to carry over the naming convention of the file headers into our application. We still want our attributes to follow snake case.

Reply:
Good eye! That was an error on my part, I just wanted to guarantee it working :)